http/BuilderSelector.cpp

```diff
@@ -20,7 +20,13 @@ AResponseBuilder* BuilderSelector::getMatchingBuilder(Request const& request) {
   if (BuilderSelector::isDirRedirectSelected(fullPath))
     return new RedirectBuilder(request, request.getPath() + '/');
 
-  return new StaticFileBuilder(request);
+  // GET -> StaticFileBuilder
+  // DELETE -> DeleteBuilder
+  if (request.getMethod() == HTTP_GET)
+    return new StaticFileBuilder(request);
+  else {
+    return new DeleteBuilder(request);
+  }
```

Comment:
else문이 빠져도 좋을 것 같아요. 그리고 주석도 위로 빼거나 지우면 좋을 것 같아요

```suggestion
  if (request.getMethod() == HTTP_GET)
    return new StaticFileBuilder(request);

  return new DeleteBuilder(request);
  
```